README.md

```diff
@@ -46,32 +47,35 @@ For convenient local development, we have included two components:
   Additionally, the network sets up a [Better Call Dev](https://github.com/baking-bad/bcdhub) instance
   for introspection of the deployed contract and its operations. The BCD interace is available in
   your browser at http://localhost:8000.
-- A script `./sandbox.sh` that sets up a identities for a local Deku cluster.
+- A script `./sandbox.sh` that sets up a identity for a local Deku cluster.
```

Comment:
This is "more wrong", it's identities, the deku cluster has multiple idendenties.